biojava-structure/src/main/java/org/biojava/nbio/structure/align/ce/CeParameters.java

```diff
@@ -418,6 +425,12 @@ public void setSubstitutionMatrix(
 	}
 
 
+	public boolean isOptimizeAlignment() {
+		return optimizeAlignment;
+	}
 
+	public void setOptimizeAlignment(boolean optimizeAlignment) {
+		this.optimizeAlignment = optimizeAlignment;
```

Comment:
Can you copy the documentation for the parameter here? Users can read it here. (same here)